tests/test_contour.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import torch
+from monai.transforms import Contour
+
+def genImg(width, height):
+    img = torch.zeros(width, height, dtype=torch.float)
+    row_idx = torch.randint(0, width, (width // 2, ))
+    col_idx = torch.randint(0, height, (height // 2, ))
+    img[row_idx, col_idx] = 255
+    return img.view(1, 1, width, height)
+    
+class TestContour(unittest.TestCase):
+    def test_contour(self):
+        width, height = 512, 512
+        test_image = genImg(width, height)
+
+        input_param = {"to_resize": False, "kernel_type": "Laplace"}
+        test_result = Contour(**input_param)(test_image)
+        self.assertEqual(test_result.shape, (width, height))
```

Comment:
please also add a test for small input data and assert the output values as expected.

Reply:
I think there's no need, as output_width is always equal to input_width when padding and stride are 1.

Reply:
Sorry, maybe I didn't make it clear, I mean to add a test to check all the values in array, for example:
input data:
```
[[0, 0, 0, 1, 1],
[0, 0, 0, 1, 1],
[0, 0, 1, 1, 1],
[0, 1, 1, 1, 1],
[1, 1, 1, 1, 1]]
```
expected output:
```
[[0, 0, 0, 1, 1],
[0, 0, 0, 1, 1],
[0, 0, 1, 0, 1],
[0, 1, 0, 0, 1],
[1, 1, 1, 1, 1]]
```
right?
use `np.testing.assert_allclose(expected, result)` to test.

Reply:
thanks Nic, it's really necessary